components/feed/feed.jsx

```diff
@@ -1,40 +1,40 @@
-import styles from '../styles/Feed.module.css';
-import { useState, useEffect, useContext } from 'react';
+import styles from '../../styles/Feed.module.css';
+import { useState, useEffect, useContext, useReducer } from 'react';
 import axios from 'axios';
 import Carousel from 'react-elastic-carousel';
-import Image from 'next/image'
-import logo from '../public/Gamut_logo_small.png'
-import firebase from '../firebase.js'
-import { getAuth } from "firebase/auth";
-import NavBar from './navbar/Nav.jsx';
-import MainContext from "../context/MainContext";
+import Image from 'next/image';
+import logo from '../../public/Gamut_logo_small.png';
+import useAuth from '../../firebase.js';
+import NavBar from '../navbar/Nav.jsx';
+import MainContext from "../../context/MainContext";
+import Lottie from './lottie.js';
+import animationDislike from './dislike.json';
+import animationConfetti from './sliderEffect.json';
 
 export default function Feed() {
   const { exploreData } = useContext(MainContext);
   const [filteredPosts, setFilteredPosts] = useState([]);
   const [votes, setVotes] = useState(0);
-  const auth = getAuth();
-  let posts = filteredPosts.length === 0 ? exploreData : filteredPosts
+  const email = useAuth();
 
-  const checkLoginStatus = () => {
-    if(auth.currentUser !== null) {
-      axios.get('http://localhost:3000/api/getPosts', {params: {email: auth.currentUser.email}})
+  useEffect(()=>{
+    if(email !== null) {
+      axios.get('http://localhost:3000/api/getPosts', {params: {email: email}})
```

Comment:
Hi, can you remove localhost:3000 from the get request , only need to include api onward.

Reply:
```
axios.get('/api/getPosts', {
  params: {
    email: email
  }
})
```

Reply:
the inclusion of causes requests to fail in deployment